src/components/Copyright/Copyright.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import RsCopyright from 'react-spatial/components/Copyright';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Nice you used MUI styling 👍

Reply:
I actually just noticed the MUI style solution is not ideal in this case, since the MUI breakpoints always refer to the window width, regardless how big the web component container is :/ Will find an alternate solution